packages/keymaps/src/browser/style/index.css

```diff
@@ -121,6 +121,10 @@
     background-color: var(--theia-editorWidget-background);
 }
 
+.kb table th:first-child {
+    width: 4.17%;
+}
+
 .kb table .th-action {
     width: 35px;
```

Comment:
I think we can modify this value to instead (it fixes the issue and does not look like a magic number):

```css

.kb table .th-action {
    width: 5%;
}
```

Reply:
While making it 5% also works, but according to the value I computed by rowLength/first-childLength the value turned out to be 4.17, however using 5% will not make much of a difference but instead rounding it to either 4.2 or 4% would be a bit more accurate, I can also add comments if this needs to be expressed more clearly

Reply:
How did you calculate `4.17` as a number?

Reply:
(Width of the row) / (Width of the column) * 100

Reply:
> (Width of the row) / (Width of the column) * 100

Do you know both what the width or the row and width of the column are?

Reply:
I used the inspect element tool and tried finding out widths on different responsive screen sizes and the ratio was constant till 2decimal places.

Reply:
I don't think we need to overthink it, something like `4%` or `5%` would be more natural.
In any case, it may be that we need to adjust other columns anyways to get the best possible visual layout for the keybindings.